lib/src/__tests__/setup.js

```diff
@@ -36,3 +36,24 @@ console.error = (...args) => {
     throw new Error(message);
   }
 };
+
+// Convert any console warning into a thrown error
+const warn = console.warn;
+console.warn = (...args) => {
+  warn.apply(console, args);
+  if (args[0] instanceof Error) {
```

Comment:
I think it will not work for warn, right?

Reply:
I don't know. In the version of the module in the main repository, we assume the arguments is an array of strings.

```jsx
  console.error = (...args) => {
    // Can't use log as karma is not displaying them.
    console.info(...args);
    throw new Error(...args);
  };

  console.warn = (...args) => {
    // Can't use log as karma is not displaying them.
    console.info(...args);
    throw new Error(...args);
  };
```

We never had any issue with it yet. But it seems that the console methods can accept any object as input. What do you want to do?